app/assets/stylesheets/custom.css.scss

```diff
@@ -225,3 +225,33 @@ a:hover, a:active {
     font: 400 18px/1 'Kaushan Script', cursive;
     color: $light-shadow-grey;
 }
+
+text {
+  font-size: 11px;
+  pointer-events: none;
+}
+
+text.parent {
+  fill: #1f77b4;
+}
+
+circle {
+  fill: #ccc;
+  stroke: #999;
+  pointer-events: all;
+}
+
+circle.parent {
+  fill: #1f77b4;
+  fill-opacity: .1;
+  stroke: steelblue;
```

Comment:
Color `steelblue` should be written in hexadecimal form as `#4682b4`<br>Color literals like `steelblue` should only be used in variable declarations; they should be referred to via variable everywhere else.